deploy/kubernetes/fluentd-sumologic.yaml.tmpl

```diff
@@ -437,7 +432,6 @@ apiVersion: v1
 kind: Service
 metadata:
   name: fluentd
-  namespace: sumologic
```

Comment:
Will the script fill these values in automatically? seems like we only added the `namespace: $NAMESPACE` for the service account

Reply:
If you look at line 189 of `setup.sh` we use `kubectl -n` so for all objects that don't have a namespace set in the YAML it will use the namespace passed in via `-n`.
The ClusterRoleBinding is different in that we need to define in which namespace the `subject` lives. So for that we keep the `subjects[].ServiceAccount.namespace` field.

Reply:
Gotcha, thanks for clarifying.

Reply:
I guess the issue is that people who don't use the setup script will have to manually replace the `$NAMESPACE` or remember to use sed before applying. Since setup is generally a one-time process I'm okay with this, and when we integrate helm this should be a non-issue.

Reply:
👍

Reply:
We should update the deployment README to specify this change though, cc @frankreno